### PR TITLE
Modify JS file to comply with the new data structure

### PR DESCRIPTION
**Pull request by godarkrai**
Modifies the JS used in the editor toolbar that inputs bracket data into the editor to comply with the new SAP API's data structure.

### CODE REVIEW

File: resources/scripts/ext.dota2WebApi.toolbar.js [hunk 1]
```diff
@@ -529,165 +475,109 @@ $( function() {
...
-				if ( data.dota2dbapi.isresult ) {
-					const result = data.dota2dbapi.result;
+				if ( data.dota2dbapi.error === undefined ) {
+					const result = data.dota2dbapi;
 
 					start += '|VOD=';
 					start += '\n';
+					// radiantPicks = "{{MatchSeries/Picks";
+					radiantPicks = result.heroVeto.radiant.picks;
+					// radiantPicks += "}}\n";
```
Comment (KilMer56) [inline resources/scripts/ext.dota2WebApi.toolbar.js:499]: Are the comments still needed?

**Review by KilMer56 — COMMENTED** [review]
Is there a page I can verify the behavior of it?

File: resources/scripts/ext.dota2WebApi.toolbar.js (hunk 1 above)
Comment (FO-nTTaX) [reply resources/scripts/ext.dota2WebApi.toolbar.js:499]: They are comments, probably not...

**Comment by FO-nTTaX** [comment]
image

We don't have team names, we might want to fix that or it's hard to use in the popup

**Comment by godarkrai** [comment]
I've removed the comments, added team names instead of ids and changed a bit of the api parsing style due to change in the structure of data

**Pull request merged**